Tests/MSFT_xWindowsOptionalFeature.Tests.ps1

```diff
@@ -0,0 +1,311 @@
+<# 
+.summary
+    Test suite for MSFT_xWindowsOptionalFeature.psm1
+#>
+[CmdletBinding()]
+param()
+
+Import-Module $PSScriptRoot\..\DSCResources\MSFT_xWindowsOptionalFeature\MSFT_xWindowsOptionalFeature.psm1
```

Comment:
yes, we've added those templates recently, so majority of our modules don't use them, but that's recommended approach. Take a look at https://github.com/PowerShell/DscResources/blob/master/TestsGuidelines.md